ilastik/workflows/tracking/conservation/animalConservationTrackingWorkflow.py

```diff
@@ -32,7 +32,7 @@ def connectLane(self, laneIndex):
            
         super(AnimalConservationTrackingWorkflowBase, self).connectLane(laneIndex) 
         
-    def _createDivisionDetectionApplet(self):
+    def _createDivisionDetectionApplet(self,selectedFeatures=None):
```

Comment:
Code calls the proc in conservationTracking, where it is set to dict() so in the current code either works.